client/src/app/shared/video/modals/video-report.component.ts

```diff
@@ -46,14 +53,82 @@ export class VideoReportComponent extends FormReactive implements OnInit {
     return ''
   }
 
+  get timestamp () {
+    return this.form.get('timestamp').value
+  }
+
+  getVideoEmbed () {
+    return this.sanitizer.bypassSecurityTrustHtml(
+      buildVideoEmbed(
+        buildVideoLink({
+          baseUrl: this.video.embedUrl,
+          title: false,
+          warningTitle: false
+        })
+      )
+    )
+  }
+
   ngOnInit () {
     this.buildForm({
-      reason: this.videoAbuseValidatorsService.VIDEO_ABUSE_REASON
+      reason: this.videoAbuseValidatorsService.VIDEO_ABUSE_REASON,
+      predefinedReasons: mapValues(VideoAbusePredefinedReasonsIn, r => null),
```

Comment:
`mapValues` -> `Object.values`?

Reply:
no, this is only a getter for values, not an iterator returning an object with the same keys mapped over a function.